lib/screens/onboarding/list_of_companies_screen.dart

```diff
@@ -63,7 +63,8 @@ class CompanyListScreen extends StatelessWidget {
                                 keyboardType: TextInputType.text,
                                 onTextFieldChanged: (value) {}),
                             const SizedBox(height: spacingMedium),
-                            CompanyList(
+                            CompaniesGridView(
```

Comment:
What is the naming convention? Why are we using CompaniesGridView without knowing if its for mobile or web?

Reply:
Initially i made 2 gridview for mobile and web but later realised that there is just a small change in both the screens so why make 2 screens.So i just made the screen with name companies gridview with toggle in it for mobile or web.